v5/api/cpp/motors.rst

```diff
@@ -1405,6 +1598,103 @@ setting ``errno``.
 
 ----
 
+set_pos_pid
+~~~~~~~~~~~
+
+Sets one of ``pros::motor_pid_s_t`` for the motor. This intended to just modify the
+main PID constants.
+
+Only non-zero values of the struct will change the existing motor constants.
+
+.. warning:: This feature is in beta, it is advised to use caution when modifying
+             the PID values. The motor could be damaged by particularly large constants.
+
+Analogous to `motor_set_pos_pid <../c/motors.html#motor-set-pos-pid>`_.
+
+.. tabs ::
+   .. tab :: Prototype
+      .. highlight:: cpp
+      ::
+
+        int32_t pros::Motor::set_pos_pid ( const motor_pid_s_t pid )
+
+   .. tab :: Example
+      .. highlight:: cpp
+      ::
+
+        #define KF 0
+        #define KP 1.0f
+        #define KI 0.001f
+        #define KD 0.1f
+
+        void initialize() {
+          pros::Motor motor (1);
+          pros::motor_pid_s_t pid = pros::Motor::convert_pid(KF, KP, KI, KD);
+          motor.set_pos_pid(pid);
+        }
+
+============ ===============================================================
+ Parameters
+============ ===============================================================
+ pid          The new motor PID constants
+============ ===============================================================
+
+**Returns:** ``1`` if the operation was successful or ``PROS_ERR`` if the operation failed,
+setting ``errno``.
+
+----
+
+set_pos_pid_full
+~~~~~~~~~~~~~~~~
+
+Sets one of ``pros::motor_pid_full_s_t`` for the motor. This intended to just modify the
+main PID constants.

```

Comment:
I can see `set_pos_pid` as "intended to just modify the main PID constants", but isn't the point of the `_full` variants to modify everything?